HDF5Examples/CMakeLists.txt

```diff
@@ -159,6 +159,7 @@ if (${H5_LIBVER_DIR} GREATER 16)
       endif ()
 
       configure_file (${H5EX_F90_SRC_DIR}/H5D/h5_version.h.in ${PROJECT_BINARY_DIR}/FORTRAN/H5D/h5_version.h @ONLY)
+      configure_file (${H5EX_F90_SRC_DIR}/H5D/h5_version.h.in ${PROJECT_BINARY_DIR}/FORTRAN/H5G/h5_version.h @ONLY)
```

Comment:
`${H5EX_F90_SRC_DIR}/H5D/h5_version.h.in` -> `${H5EX_F90_SRC_DIR}/H5G/h5_version.h.in`

Reply:
fixed